dhall/tests/format/multilineTrailingWhitespaceB.dhall

```diff
@@ -0,0 +1,7 @@
+{ x =
+    ''
+      
```

Comment:
There's trailing whitespace on this line, but GitHub doesn't seem to display it unlike the trailing whitespace after `foo` on line 5.